attestation_demo/AppInitiator/App.cpp

```diff
@@ -60,8 +60,8 @@
 
 #define REMOTE_ADDR "127.0.0.1"
 #define REMOTE_PORT "10601"
-#define PELZ_REQ_ENC_SIGNED 3
-#define PELZ_REQ_DEC_SIGNED 4
+#define PELZ_REQ_ENC_PROTECTED 5
```

Comment:
So far, the attestation demo client hasn't been directly using any pelz source code.
The initial reason for this was the idea that pelz clients would be part of fully standalone applications compiled separately outside of the pelz repository, and that keeping them separate would outweigh the benefits of direct code reuse.

The underlying attestation and communication protocol code comes from libsgx, so it's not being duplicated between pelz and the client.
The standard request and response messages are essentially a straightforward and documented API, so it seemed reasonable to implement them without reusing the pelz enums and other code, especially because the demo client doesn't need to support every possible type of message or every pelz configuration.

However, adding the new request type and the new key derivation step makes the interface less straightforward, so it would make more sense to reuse code there.
It would be preferable to do this through a new "pelz client library" so the client could access pelz code without directly using the pelz repo.